src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,115 @@
+package controller;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import model.bonusball.BonusBall;
+import model.lotto.LottoCount;
+import model.lotto.LottoStorage;
+import model.lottonumber.LottoNumber;
+import model.result.Rank;
+import model.result.RateOfReturn;
+import model.winningnumber.LottoWinningNumber;
+import utils.InputValidateUtils;
+import view.InputView;
+import view.OutputView;
+
+public class LottoController {
+	private static final String LOTTO_COUNT_BLANK_ERROR_MESSAGE = "[Error]: 금액을 입력해주세요.";
+	private static final String LOTTO_COUNT_NUMBER_ERROR_MESSAGE = "[Error]: 금액은 숫자를 입력해주세요.";
+	private static final String WINNING_NUMBER_ERROR_MESSAGE = "[Error]: 당첨 번호는 숫자여야 합니다.";
+	private static final String WINNING_NUMBER_BLANK_ERROR_MESSAGE = "[Error]: 당첨 번호를 입력하세요.";
+	private static final String BONUS_BALL_BLANK_ERROR_MESSAGE = "[Error]: 보너스 볼을 입력해주세요.";
+	private static final String BONUS_BALL_NUMBER_ERROR_MESSAGE = "[Error]: 보너스 볼은 숫자여야 합니다.";
+	private static final String DELIMITER_COMMA = ",";
+
+	private final InputView inputView = new InputView();
+	private final OutputView outputView = new OutputView();
+
+	private LottoStorage lottoStorage;
+	private LottoWinningNumber lottoWinningNumber;
+	private BonusBall bonusBall;
+	private RateOfReturn rateOfReturn;
+
+	public void playGame() {
+		makeLottos();
+		storeWinningNumber();
+		storeBonusBall();
+		compareLottoWithWinningNumber();
+		showResult();
+	}
+
+	private void makeLottos() {
+		try {
+			String money = inputView.inputMoney();
+			InputValidateUtils.inputBlankAndNumber(money, LOTTO_COUNT_BLANK_ERROR_MESSAGE,
+				LOTTO_COUNT_NUMBER_ERROR_MESSAGE);
+			storeMoneyInRateOfReturn(Integer.parseInt(money));
+			lottoStorage = new LottoStorage(new LottoCount(Integer.parseInt(money)));
+			outputView.printLottos(lottoStorage.getLottoStorageDTO());
+		} catch (Exception e) {
+			outputView.printErrorMessage(e.getMessage());
+			makeLottos();
+		}
+	}
```

Comment:
재귀호출로 해결하시는 거라면 아래와 같은 방식은 어떨까요?

```suggestion
        private LottoStorage makeLottos() {
		try {
			String money = inputView.inputMoney();
			InputValidateUtils.inputBlankAndNumber(money, LOTTO_COUNT_BLANK_ERROR_MESSAGE,
				LOTTO_COUNT_NUMBER_ERROR_MESSAGE);
			storeMoneyInRateOfReturn(Integer.parseInt(money));
			LottoStorage lottoStorage = new LottoStorage(new LottoCount(Integer.parseInt(money)));
                        outputView.printLottos(lottoStorage.getLottoStorageDTO());
                        return lottoStorage;
		} catch (Exception e) {
			outputView.printErrorMessage(e.getMessage());
			return makeLottos();
		}
	}
```

Reply:
혹시 이렇게 하는 방식의 이점이 있나요?